Examples/Tests/circuits/inputs_silicon

```diff
@@ -0,0 +1,101 @@
+############ FIELDS #############
+#################################
+
+my_constants.pi = 3.14159265359
+my_constants.L = 141.4213562373095e-6
+my_constants.c = 299792458.
+my_constants.wavelength = 4.e-6
+
+my_constants.sigma_0 = 0.0
+my_constants.sigma_silicon = 1.e-4
+my_constants.epsilon_r_silicon = 11.68
+my_constants.epsilon_0 = 8.8541878128e-12
+my_constants.mu_0 = 1.25663706212e-06
+my_constants.mu_r_silicon = 1.0
+
+my_constants.TP = 2.5e-14
+
+warpx.E_ext_grid_init_style = parse_E_ext_grid_function
+warpx.Ex_external_grid_function(x,y,z) = "0."
+#warpx.Ex_external_grid_function(x,y,z) = "1.e5*exp(-(y)**2/(L/32)**2)*cos(2*pi*(y)/wavelength) * (x < 0.000004) * (x > -0.000004) * (z < 0.000004) * (z > -0.000004)"
+#warpx.Ex_external_grid_function(x,y,z) = "1.e5*exp(-(y)**2/(L/32)**2)*cos(2*pi*(y)/wavelength)"
+warpx.Ey_external_grid_function(x,y,z) = "0."
+warpx.Ez_external_grid_function(x,y,z) = "0."
+
+warpx.B_ext_grid_init_style = parse_B_ext_grid_function
+warpx.Bx_external_grid_function(x,y,z) = "0."
+warpx.By_external_grid_function(x,y,z) = "0."
+warpx.Bz_external_grid_function(x,y,z) = "0."
+#warpx.Bz_external_grid_function(x,y,z) = "(-1.e5*exp(-(y)**2/(L/32)**2)*cos(2*pi*(y)/wavelength)/c) * (z < 0.000004) * (z > -0.000004) * (x < 0.000004) * (x > -0.000004)"
+#warpx.Bz_external_grid_function(x,y,z) = "(-1.e5*exp(-(y)**2/(L/32)**2)*cos(2*pi*(y)/wavelength)/c)"
+
+warpx.E_excitation_on_grid_style = parse_E_excitation_grid_function
+
+warpx.Ex_excitation_flag_function(x,y,z) = "(x >= -0.000004 - (1.25e-07)) * (x <= 0.000004 + (1.25e-07)) * (z >= 0.000004 - (1.25e-07)) * (z <= 0.000004 + (1.25e-07))"
+
+warpx.Ey_excitation_flag_function(x,y,z) = "(x >= 0.000004 - (1.25e-07)) * (x <= 0.000004 + (1.25e-07)) * (z >= -0.000004 - (1.25e-07)) * (z <= 0.000004 + (1.25e-07)) + (x >= -0.000004 - (1.25e-07)) * (x <= -0.000004 + (1.25e-07)) * (z >= -0.000004 - (1.25e-07)) * (z <= 0.000004 + (1.25e-07)) + (x >= -0.000004 - (1.25e-07)) * (x <= 0.000004 + (1.25e-07)) * (z >= 0.000004 - (1.25e-07)) * (z <= 0.000004 + (1.25e-07))"
+
+warpx.Ez_excitation_flag_function(x,y,z) = "(x >= 0.000004 - (1.25e-07)) * (x <= 0.000004 + (1.25e-07)) * (z >= -0.000004 - (1.25e-07)) * (z <= 0.000004 + (1.25e-07)) + (x >= -0.000004 - (1.25e-07)) * (x <= -0.000004 + (1.25e-07)) * (z >= -0.000004 - (1.25e-07)) * (z <= 0.000004 + (1.25e-07))"
+
+# turns off PEC walls
+#warpx.Ex_excitation_flag_function(x,y,z) = "0."
+#warpx.Ey_excitation_flag_function(x,y,z) = "0."
+#warpx.Ez_excitation_flag_function(x,y,z) = "0."
+
+warpx.Ex_excitation_grid_function(x,y,z,t) = ".0"
+warpx.Ey_excitation_grid_function(x,y,z,t) = ".0"
+warpx.Ez_excitation_grid_function(x,y,z,t) = ".0"
+
+warpx.B_excitation_on_grid_style = parse_B_excitation_grid_function
+
+#warpx.Bx_excitation_flag_function(x,y,z) = "0."
+warpx.Bx_excitation_flag_function(x,y,z) = "(x > -0.000004) * (x < 0.000004) * (z > -0.000004) * (z < 0.000004) * (y <= -16.e-6 + 1.25e-7) * (y >= -16.e-6 - 1.25e-7)"
```

Comment:
in case users need to modify these numbers, would a my_constants definition will help to avoid typos